Packs/VMwareWorkspaceONEUEM/Integrations/VMwareWorkspaceONEUEM/VMwareWorkspaceONEUEM.py

```diff
@@ -0,0 +1,644 @@
+        Capitalize only the first letter of a string
+        :param string: string whose first letter needs to be capitalized
+        :return: string with first letter capitalized
+        """
+        return string[0].upper() + string[1:]
+
+    if isinstance(src, list):
+        return_src = [camel_to_pascal(obj) for obj in src]
+        return return_src
+
+    for key, value in src.items():
+        if isinstance(value, (dict, list)):
+            return_src[capitalize_first_letter(key)] = camel_to_pascal(value)  # type: ignore
+        else:
+            return_src[capitalize_first_letter(key)] = value
+    return return_src
+
+
+def prepare_context_hr_os_updates_list_command(result: dict, uuid: str) -> Tuple[Union[dict, List[dict]], str]:
+    """
+    To prepare context and human readable output for vmwuem_device_os_updates_list_command.
+
+    :type result: ``dict``
+    :param result: dictionary returned by api response of vmwuem_device_os_updates_list_command.
+
+    :type uuid: ``str``
+    :param uuid: argument of vmwuem_device_os_updates_list_command.
+
+    :return: Context and human readable output.
+    :rtype: ``Tuple[Dict, str]``
+    """
+    result = remove_empty_elements_for_context(result)  # type: ignore
+    result['OSUpdateList'] = camel_to_pascal(result['OSUpdateList'])
+    result['Uuid'] = uuid.lower()
+    context_data = result
+    hr = []
+    for osupdate in result['OSUpdateList']:
+        release_date = osupdate.get('ReleaseDate', '')
+        expiration_date = osupdate.get('ExpiationDate', '')
+        if release_date:
+            release_date = dateparser.parse(release_date).strftime(READABLE_DATE_FORMAT)  # type: ignore
+        if expiration_date:
+            expiration_date = dateparser.parse(expiration_date).strftime(READABLE_DATE_FORMAT)  # type: ignore
+        data = {
+            'Device UUID': result['Uuid'],
+            'Update Name': osupdate.get('DeviceUpdateName', ''),
+            'Update Version': osupdate.get('DeviceUpdateVersion', ''),
+            'Critical Update': "Yes" if osupdate.get('IsCritical', False) else "No",
+            'Restart Required': "Yes" if osupdate.get('RestartRequired', False) else "No",
+            'Release Date': release_date,
+            'Expiration Date': expiration_date
+        }
+        hr.append(data)
+
+    headers = ['Device UUID', 'Update Name', 'Update Version', 'Critical Update', 'Restart Required',
+               'Release Date', 'Expiration Date']
+    hr_output = tableToMarkdown('OSUpdate(s)', hr, headers=headers, removeNull=True)
+
+    return context_data, hr_output
+
+
+def strip_args(args: dict):
+    """
+    Strips argument dictionary values of spaces
+
+    :type args: dict
+    :param args: argument dictionary
+    """
+    for key, value in args.items():
+        if isinstance(value, str):
+            args[key] = value.strip()
+
+
+def is_present_in_list(value_to_check: Any, list_to_check_in: List[Any], message: str) -> Optional[bool]:
```

Comment:
No need for this command. You can check this manually where you use it. 
We also give default values here.

Reply:
@Shellyber , it's an internal helper method that we used to avoid the higher cognitive complexity of the code.